debian/rules

```diff
@@ -8,6 +8,10 @@ export PYBUILD_NAME=eos-data-distribution
 
 override_dh_install:
 	dh_install --fail-missing
+	mkdir -p debian/edd-avahi-monitor/usr/bin
```

Comment:
No, normally you would just put the files you want in the `$package.install` file and debhelper takes care of distributing them. In fact, I don't even know how the commands below would succeed. `dh_install` shouldn't be putting those files in the python-eos-data-distribution package since they're not listed in its install file.

The normal way to do this would be to put `usr/bin/edd-avah-monitor` in `edd-avahi-monitor.install` and `usr/bin/edd-usb-producer` in `edd-usb-producer.install`.